tests/Unit/Momento.Sdk.Tests/Responses/CacheGetResponseTest.cs

```diff
@@ -14,16 +14,15 @@ public void CorrectResultMapping()
         string cacheBody = "test body";
         ByteString body = ByteString.CopyFromUtf8(cacheBody);
         _GetResponse serverResponseHit = new _GetResponse() { CacheBody = body, Result = ECacheResult.Hit };
-        CacheGetResponse responseHit = new CacheGetResponse(serverResponseHit);
+        CacheGetResponse.Hit responseHit = new CacheGetResponse.Hit(serverResponseHit);
         Assert.Equal(CacheGetStatus.HIT, responseHit.Status);
         Assert.Equal(cacheBody, responseHit.String());
 
-        _GetResponse serverResponseMiss = new _GetResponse() { Result = ECacheResult.Miss };
-        CacheGetResponse responseMiss = new CacheGetResponse(serverResponseMiss);
-        Assert.Equal(CacheGetStatus.MISS, responseMiss.Status);
-
-        _GetResponse serverResponseBadRequest = new _GetResponse() { Result = ECacheResult.Invalid };
-        _ = Assert.Throws<InternalServerException>(() => new CacheGetResponse(serverResponseBadRequest));
+        // _GetResponse serverResponseMiss = new _GetResponse() { Result = ECacheResult.Miss };
+        // CacheGetResponse.Miss responseMiss = new CacheGetResponse.Miss(serverResponseMiss);
+        // Assert.Equal(CacheGetStatus.MISS, responseMiss.Status);
 
+        // _GetResponse serverResponseBadRequest = new _GetResponse() { Result = ECacheResult.Invalid };
+        // _ = Assert.Throws<InternalServerException>(() => new CacheGetResponse.Success(serverResponseBadRequest));
```

Comment:
intentional?

Reply:
Yes, those are obsolete due to class restructuring. The response subtypes are tested in the integration tests, so we're not losing coverage.